accessiblity.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: page
+title: Accessibility
+
+permalink: /accessibility/
+---
+
+Flutter is committed to enable developers to make their Flutter apps accessible:
+usable for all users, including users with disabilities such as blindness or
+motor impairment.
+
+* Placeholder for TOC
+{:toc}
+
+# Flutter's accessibility support
+
+Flutter supports three components of accessibility support:
+
+* **[Large fonts](#large-fonts)**: Render text widgets with user specified font
+  font sizes
+* **[Screen readers](#screen-reader)**: Communicate spoken feedback about UI
+  contents
+* **[High contrast](#high-contrast)**: Render widgets with colors that have
```

Comment:
Should it be called "high contrast" or "sufficient contrast"?  You should pick one.  I think the accessibility team prefers "sufficient contrast".

Reply:
Changed to 'sufficient'